.cirrus.yml

```diff
@@ -0,0 +1,70 @@
+env:
+  intermediateArtifactName: dotnet-$CIRRUS_BUILD_ID.tar.gz
+
+Linux_task:
+  container:
+    image: ubuntu:xenial
+
+  packages_script: |
+    apt update
+    apt install -y libunwind8 libicu55 curl git clang-3.9 cmake make libc6-dev libssl-dev \
+                   libkrb5-dev libcurl4-openssl-dev zlib1g-dev liblldb-3.9-dev gettext \
+                   libicu-dev liblttng-ust-dev libkrb5-dev git
+
+  checkout repos_script: ./scripts/checkout.sh
+
+  build_script: |
+    cd $CIRRUS_WORKING_DIR/artifacts/sources/coreclr
+    bash -x ./build.sh -skiptests -skipmanagedtools -skipnative  -skipcrossgen -release -osgroup FreeBSD \
+        /p:osGroup=FreeBSD /p:PackageRid=freebsd-x64
+    cd $CIRRUS_WORKING_DIR/artifacts/sources/corefx
+    ./build.sh -c Release --restore
+#    ./build.sh -c Release --build  /p:osGroup=FreeBSD /p:PackageRid=freebsd-x64
```

Comment:
@wfurt, I have commented this line (corefx managed crossgen for FreeBSD on Linux) because it was failing with:

```sh
Build FAILED.
/tmp/cirrus-ci-build/artifacts/sources/corefx/external/runtime/runtime.depproj : error NU1102: Unable to find package transport.runtime.freebsd-x64.Microsoft.NETCore.Runtime.CoreCLR with version (>= 3.0.0-rc2.19462.5) [/tmp/cirrus-ci-build/artifacts/sources/corefx/external/dir.proj]
/tmp/cirrus-ci-build/artifacts/sources/corefx/external/runtime/runtime.depproj : error NU1102:   - Found 153 version(s) in dotnet-core [ Nearest version: 3.0.0-preview1-27131-02 ] [/tmp/cirrus-ci-build/artifacts/sources/corefx/external/dir.proj]
/tmp/cirrus-ci-build/artifacts/sources/corefx/external/runtime/runtime.depproj : error NU1102:   - Found 51 version(s) in dotnet-coreclr [ Nearest version: 3.0.0-preview-27525-73 ] [/tmp/cirrus-ci-build/artifacts/sources/corefx/external/dir.proj]
/tmp/cirrus-ci-build/artifacts/sources/corefx/external/runtime/runtime.depproj : error NU1102:   - Found 0 version(s) in nuget.org [/tmp/cirrus-ci-build/artifacts/sources/corefx/external/dir.proj]
/tmp/cirrus-ci-build/artifacts/sources/corefx/external/runtime/runtime.depproj : error NU1102:   - Found 0 version(s) in darc-pub-dotnet-standard-a5b5f2e [/tmp/cirrus-ci-build/artifacts/sources/corefx/external/dir.proj]
/tmp/cirrus-ci-build/artifacts/sources/corefx/external/runtime/runtime.depproj : error NU1102: Unable to find package runtime.freebsd-x64.Microsoft.NETCore.TestHost with version (>= 3.0.0-rc2.19462.5) [/tmp/cirrus-ci-build/artifacts/sources/corefx/external/dir.proj]
/tmp/cirrus-ci-build/artifacts/sources/corefx/external/runtime/runtime.depproj : error NU1102:   - Found 153 version(s) in dotnet-core [ Nearest version: 3.0.0-preview1-27131-02 ] [/tmp/cirrus-ci-build/artifacts/sources/corefx/external/dir.proj]
/tmp/cirrus-ci-build/artifacts/sources/corefx/external/runtime/runtime.depproj : error NU1102:   - Found 51 version(s) in dotnet-coreclr [ Nearest version: 3.0.0-preview-27525-73 ] [/tmp/cirrus-ci-build/artifacts/sources/corefx/external/dir.proj]
/tmp/cirrus-ci-build/artifacts/sources/corefx/external/runtime/runtime.depproj : error NU1102:   - Found 0 version(s) in nuget.org [/tmp/cirrus-ci-build/artifacts/sources/corefx/external/dir.proj]
/tmp/cirrus-ci-build/artifacts/sources/corefx/external/runtime/runtime.depproj : error NU1102:   - Found 0 version(s) in darc-pub-dotnet-standard-a5b5f2e [/tmp/cirrus-ci-build/artifacts/sources/corefx/external/dir.proj]
/tmp/cirrus-ci-build/artifacts/sources/corefx/external/runtime/runtime.depproj : error NU1102: Unable to find package transport.runtime.freebsd-x64.Microsoft.NETCore.Jit with version (>= 3.0.0-rc2.19462.5) [/tmp/cirrus-ci-build/artifacts/sources/corefx/external/dir.proj]
/tmp/cirrus-ci-build/artifacts/sources/corefx/external/runtime/runtime.depproj : error NU1102:   - Found 153 version(s) in dotnet-core [ Nearest version: 3.0.0-preview1-27131-02 ] [/tmp/cirrus-ci-build/artifacts/sources/corefx/external/dir.proj]
/tmp/cirrus-ci-build/artifacts/sources/corefx/external/runtime/runtime.depproj : error NU1102:   - Found 51 version(s) in dotnet-coreclr [ Nearest version: 3.0.0-preview-27525-73 ] [/tmp/cirrus-ci-build/artifacts/sources/corefx/external/dir.proj]
/tmp/cirrus-ci-build/artifacts/sources/corefx/external/runtime/runtime.depproj : error NU1102:   - Found 0 version(s) in nuget.org [/tmp/cirrus-ci-build/artifacts/sources/corefx/external/dir.proj]
/tmp/cirrus-ci-build/artifacts/sources/corefx/external/runtime/runtime.depproj : error NU1102:   - Found 0 version(s) in darc-pub-dotnet-standard-a5b5f2e [/tmp/cirrus-ci-build/artifacts/sources/corefx/external/dir.proj]
```

do you know what should be correct order?

Reply:
I don't think we will be able to use Nuget at the point when bootstrap builds. Maybe we can fake PackageRID but `/p:osGroup=FreeBSD`  must stay to pick correct sources.

Reply:
Great, dropping `/p:PackageRid=freebsd-x64` did the trick.